### ISSUE TITLE
OS_FileSysAddFixedMap missing functional test

### ISSUE THREAD

**Issue by skliper** [nasa/osal#379]
**Is your feature request related to a problem? Please describe.**
Missing functional test for OS_FileSysAddFixedMap

**Describe the solution you'd like**
Add test

**Describe alternatives you've considered**
None

**Additional context**
Certification issue

**Requester Info**
Jacob Hageman - NASA/GSFC

**Comment by yammajamma** [comment]
I can take this one.

**Comment by jphickey** [comment]
FYI - as part of my upcoming changes (#449 #460), the unit tests _will_ be using this function to set up for a BSP-agnostic file map.  

I'm not sure if there is anything more that can be done with them beyond this, because any test would require going around the OSAL abstraction in a platform-dependent manner.  Furthermore, if they didn't work right, all the tests following that depend on the map will fail.

So in other words I think its covered already

**Comment by skliper** [comment]
@yammajamma - could you review the unit test updates in #461 and ensure it meets the functional test standards (covers everything we've discussed a functional test needs to cover)?  If so we can close this.  If not let's add whatever is missing and/or create a unique test.

**Comment by yammajamma** [comment]
@skliper Yes, I can take a look at that, thanks.
